app/experimenter/experiments/constants/nimbus.py

```diff
@@ -419,6 +442,7 @@ class EmailType(models.TextChoices):
     TARGETING_CHANNEL = 'browserSettings.update.channel == "{channel}"'
 
     TARGETING_CONFIGS = {
+        TARGETING_NO_TARGETING.slug: TARGETING_NO_TARGETING,
```

Comment:
I forget why we have this dict and the choices below it, we should only need one I think...  Not for this PR I mean, just something to maybe cleanup later.